Integrations/integration-Cloaken.yml

```diff
@@ -0,0 +1,79 @@
+description: 'Unshorten urls through a tor proxy server. Unshorten urls onsite with
+  the power of Tor to prevent leaking ip addresses to adversaries.  Created by CypherInt.  Visit
+  us at https://cypherint.com.  Subscribe on the Amazon marketplace: https://aws.amazon.com/marketplace/'
+detaileddescription: cloaken will return the unshortened version of a url
+configuration:
+- display: Username
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Server URL
```

Comment:
Please move the server URL to be above the credentials, and add default value of `https://cloaken.cypherint.com`

Reply:
This will be available on the amazon marketplace for purchase.  We don't want people using our free version. It is there for you to use, though.

Reply:
Not sure I follow, you want to remove the URL parameter ?